client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -123,6 +132,26 @@ export const getTotalPriceBreakdown = ( state, orderId, siteId = getSelectedSite
 		: null;
 };
 
+export const getCountriesData = ( state, orderId, siteId = getSelectedSiteId( state ) ) => {
+	if ( ! isLoaded( state, orderId, siteId ) ) {
+		return null;
+	}
+
+	const shippingLabel = getShippingLabel( state, orderId, siteId );
+	const { countriesData } = shippingLabel.storeOptions;
+	if ( isEnabled( 'woocommerce/extension-wcservices/international-labels' ) || ! countriesData ) {
```

Comment:
Why are we checking for `! countriesData` here? What will happen when the feature flag is enabled for everyone?

Reply:
That's just a shortcut so if `countriesData` is `null/undefined`, it just returns it as-is. The idea with this function was to filter the list of countries returned by the server, but if there's no list yet, there's nothing to filter.

Anyway, this logic is all rewritten in the `/v3/continents` PR. This whole function is removed.